Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -621,7 +621,7 @@ public async Task Init(IServiceCollection serviceCollection)
             string contentRoot = ServerConfigurationManager.Configuration.DashboardSourcePath;
             if (string.IsNullOrEmpty(contentRoot))
             {
-                contentRoot = Path.Combine(ServerConfigurationManager.ApplicationPaths.ApplicationResourcesPath, "jellyfin-web", "src");
+                contentRoot = Path.Combine(ServerConfigurationManager.ApplicationPaths.WebPath, "src");
```

Comment:
this `src` should be included into `ApplicationPaths.WebPath`

Reply:
I was treating it as the path to the root of the `jellyfin-web` repo, but if you think the `src` is valid too I can do so.

Reply:
In the end it might need to be `dist`, so it should be included.

Reply:
I think we should be treating `jellyfin-web/src` as product of "building" web UI (which for now is just checking it out). When webpack finally gets in we'll start having real builds of web UI (no quotes), and they shouldn't contain `src` in them any longer.

Reply:
Done.